recipe/build.sh

```diff
@@ -9,11 +9,30 @@ if [[ "${target_platform}" == osx-* ]]; then
   export CXXFLAGS="${CXXFLAGS} -DTARGET_OS_OSX=1"
 fi
 
+# For builds that are cross-compiled (aarch64), we need to build a bootstrap system first.
+# https://www.erlang.org/doc/installation_guide/install-cross#Build-and-Install-Procedure_Building-With-configuremake-Directly_Building-a-Bootstrap-System
+if [[ "${CONDA_BUILD_CROSS_COMPILATION}" -eq 1 ]]; then
+
+  BOOTSTRAP_PREFIX="${PREFIX}/../bootstrap"
+  CFLAGS= LDFLAGS= ./configure \
+      --enable-bootstrap-only \
+      --host="${CONDA_TOOLCHAIN_BUILD}" \
```

Comment:
Think we may want...

```suggestion
      --build="${BUILD}" \
      --host="${HOST}" \
```

For [example]( https://github.com/conda-forge/hdf5-feedstock/blob/cfbef1378b64f8a7bc054d5a46094ea18ebe7942/recipe/build.sh#L98-L99 )

Reply:
It is in fact critical that `--host` is the same as `--build` here, otherwise configure will fail with:

```
error: Cannot both cross compile and build a bootstrap system
```

I've added a comment.